src/renderer/Core/I18n/getCoreResources.ts

```diff
@@ -235,6 +236,7 @@ export const getCoreResources = (): { namespace: string; resources: Resources<Tr
                     automaticRescan: "Automatisches Neuscannen",
                     fuzziness: "Fuzziness",
                     maxResultLength: "Maximale Anzahl Suchergebnisse",
+                    // TODO: maxResultLengthEmptySearchTerm: "",
```

Comment:
Unfortunately I don't speak German, but happy to add in here if you can help translate!